apps/nowcasting-app/components/charts/remix-line.tsx

```diff
@@ -345,6 +349,46 @@ const RemixLine: React.FC<RemixLineProps> = ({
               />
             </>
           )}
+
+          <Line
```

Comment:
What are these two new lines?

Reply:
thanks for spotting these. the order of the lines getting slotted into the graph dictates how the lines appear. so I'll delete one set of the lines.